src/Silex/Component/Security/Http/Firewall/JWTListener.php

```diff
@@ -68,8 +68,8 @@ public function handle(GetResponseEvent $event)
 
                 return;
 
-            } catch (HttpEncodingException $e) {
-            } catch (\UnexpectedValueException $e) {
+            } catch (\Exception $e) {
+                throw $e;
```

Comment:
This exception is correctly, need return 401 http code,

No need to throw an exception on.